# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../deluge/boost.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- setup.py.orig	2007-03-25 11:58:07.000000000 +0200
+++ ./setup.py		2007-03-25 11:58:18.000000000 +0200
@@ -87,7 +87,7 @@
                     			'/usr/include/python' + pythonVersion],
                     libraries = ['boost_filesystem', 'boost_date_time',
 					'boost_program_options', 'boost_regex',
-					'boost_serialization', 'boost_thread', 
+					'boost_serialization', 'boost_thread-mt', 
 					'z', 'pthread'],
                     extra_compile_args = ["-Wno-missing-braces"],
                     sources = ['src/deluge_core.cpp',
